SU-HSDO/suhumsci
# feat(hackday): Lando docs update and Drupal admin fixes

# READY FOR REVIEW

## Summary

Made some visual improvements to admin interfaces using the `humsci_basic` theme.

![Screen Shot 2020-05-29 at 5 24 21 PM](https://user-images.githubusercontent.com/1256329/83307217-373a6780-a1d2-11ea-9c79-0a02f2d1ad7c.png)
![layout-builder-fixes](https://user-images.githubusercontent.com/1256329/83307218-386b9480-a1d2-11ea-8159-dc9b374ac906.png)

I also added a couple of lando docs things.

## Need Review By (Date)
Whenever

## Urgency
low

## Steps to Test
1. Browse to a page using a Textarea paragraph and confirm that the margin matches the "After" screenshot above.
2. Browse to a basic page, click "layout" to go to layout builder, hover over the "Main Navigation Block" item in the right sidebar and click the pencil icon to adjust the settings, and compare the interface of the accordions to confirm that it matches the "After" screenshot above.

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Sparkbox PR Checklist](../docs/SparkboxPRChecklist.md)
